### DISCUSSION TITLE
Link Announcer not working in channel  after using .optout

### DISCUSSION THREAD

**Discussion by cloneman** [TotallyNotRobots/CloudBot#480] [Q&A] [answered]
My link announcer plugin has stopped working for 1 particular channel, and this happened shortly after messing with .optout, although this may be unrelated.

In any case, there should be some way of resetting the default settings for a channel.

**Comment by linuxdaemon** [comment, marked as answer]
There is a .clearoptout command to clear a channel's optout configuration
